Tests/swift-sdk-swift-tests/IterableInboxViewControllerTests.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  IterableInboxViewControllerTests.swift
+//  swift-sdk-swift-tests
+//
+//  Created by Jay Kim on 6/4/19.
+//  Copyright © 2019 Iterable. All rights reserved.
+//
+
+import XCTest
+
+@testable import IterableSDK
+
+class IterableInboxViewControllerTests: XCTestCase {
+    func testInitializers() {
+        let inboxViewController1 = IterableInboxViewController()
+        XCTAssertEqual(inboxViewController1.tableView.numberOfSections, 1)
```

Comment:
Not sure if we need to check the number of sections for all these cases.

Reply:
Yeah, I wasn't entirely sure the way to do this, but my objective for those checks was to ensure it actually created the full view controller. It's probably not necessary though, you're right. I can just check if the VC isn't nil.